### PR TITLE
feat(list/bullet): init component

### PR DESCRIPTION
**Pull request by JoanClaret**
init `sui-list-bullet` component

image

### CODE REVIEW

File: components/list/bullet/src/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <div className={listBulletClass}>
+      {listItems &&
+        listItems.map((item, index) => (
+          <div key={index} className="sui-ListBullet-item">
+            <img
+              className="sui-ListBullet-illustration"
+              src={item.illustration}
+            />
+            <div className="sui-ListBullet-description">
+              {item.title && (
```
Comment (tureey) [inline components/list/bullet/src/index.js]: Related to the question below, making the shape `isRequired`, does it make all the `keys` required?
If so, we do not need this check

In negative case, we should think the make `illustration` and `title` required IMHO

File: components/list/bullet/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@schibstedspain/sui-list-bullet",
+  "version": "1.8.0",
```
[the remark this answers is not in the document]
Comment (JoanClaret) [reply components/list/bullet/package.json]: aww thank you!

File: components/list/bullet/src/index.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ListBullet.propTypes = {
+  /**
+   * List of bullet items
+   */
+  listItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      illustration: PropTypes.string,
+      title: PropTypes.string,
+      description: PropTypes.string
+    }).isRequired
```
[the remark this answers is not in the document]
Comment (JoanClaret) [reply components/list/bullet/src/index.js]: nop, requiered are`illustration` and `description`.
fixed!

File: components/list/bullet/src/index.js (hunk 1 above)
Comment (JoanClaret) [reply components/list/bullet/src/index.js]: fixed!
> requiered are`illustration` and `description`.

File: components/list/bullet/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+$ npm install @schibstedspain/sui-list-bullet --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import ListBullet from '@schibstedspain/sui-list-bullet'
+
+const listItems = [
```
Comment (jcger) [inline components/list/bullet/README.md]: check out this recommendation https://github.schibsted.io/scmspain/frontend-all-wiki/wiki/How-To-Pass-Wrapped-Nodes-As-Props

Comment (tureey) [reply components/list/bullet/README.md]: La verdad es que está muy bien explicado. Una de las dudas que tenía de cuando usar los `componentes compuestos`

Comment (JoanClaret) [reply components/list/bullet/README.md]: this array is just information to fill the component tags. to be honest, i don't really understand how to match this with "passing wrapped nodes". 
🤷‍♂️

Comment (JoanClaret) [reply components/list/bullet/README.md]: answering my own 🔝 question, would be: creating nodes to iterate them using the array's information.
^ _ ^
@jcger component refactored. take a look!

File: demo/list/bullet/playground [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      'Resuelve rápidamente las dudas de los interesados en tu anuncio en el chat'
+  }
+]
+
+return (
+  <div>
+    <h2>Standard</h2>
+    <ListBullet listItems={listItems}/>
+    <h2>Responsive</h2>
+    <ListBullet listItems={listItems} responsive />
```
Comment (jcger) [inline demo/list/bullet/playground]: as you can use every exported object from your src/index.js, you should we able to create something like

```jsx
<ListBullet>
  <Bullet illustration="https://s.ccdn.es/images/motivation-search.svg" title="Publica con total seguridad" text="Te damos todas las herramientas para un proceso de venta seguro" />
  <Bullet illustration="https://s.ccdn.es/images/motivation-trust.svg" title="Gestiona tu anuncio según tus necesidades" text="Edita y renueva tu anuncio tantas veces como necesites" />
  <Bullet illustration="https://s.ccdn.es/images/motivation-security.svg" title="Chatea con los compradores potenciales" text="Resuelve rápidamente las dudas de los interesados en tu anuncio en el chat" />
</ListBullet>
```

File: components/list/bullet/README.md (hunk 4 above)
Comment (jcger) [reply components/list/bullet/README.md]: you would need to update this example

File: demo/list/bullet/playground (hunk 5 above)
Comment (JoanClaret) [reply demo/list/bullet/playground]: As we spoke, i don't see the improvement decoupling a component logic that already has a coupled styles. i would see the improvement iterating isolated components.
anyways, thanks for taking time to explaining this pattern. i would use it for sure.

**Pull request merged**